src/transformers/models/realm/modeling_realm.py

```diff
@@ -1737,11 +1742,21 @@ def __init__(self, config, retriever=None):
         self.post_init()
 
     @property
-    def beam_size(self):
+    def searcher_beam_size(self):
```

Comment:
This breaks backward compatibility here - `realm.beam_size` would now error out. Can't we just leave the name?

Reply:
That's the only thing which would be nice to revert - the rest looks good to me!